libs/rest/rest/src/main/kotlin/net/corda/rest/authorization/AuthorizingSubject.kt

```diff
@@ -10,6 +12,8 @@ interface AuthorizingSubject {
      */
     val principal: String
 
+    val expiryStatus: PasswordExpiryStatus?
```

Comment:
I would prefer to have this property as non-nullable like so:
```suggestion
    val expiryStatus: PasswordExpiryStatus
```
since `AuthorizingSubject` is created only after authentication is done `expiryStatus` values should only ever be either `ACTIVE` or `CLOSE_TO_EXPIRY`. Having `null` value makes things a bit ambiguous IMO.

Reply:
I was using null as a default value where the value was required but not used. I have replaced default values with `ACTIVE`